modules/ROOT/pages/partner-manager-architecture.adoc

```diff
@@ -1,2 +1,62 @@
 = Anypoint Partner Manager Architecture
 
+With Anypoint Partner Manager, you can deploy B2B Integration message flows with your partners, either to your own hosted Mule runtimes or to MuleSoft hosted CloudHub iPaaS platform.
+
+The functional features and user experience in configuring B2B Integrations with Partner Manager are deployment-target agnostic; however, there are differences in the underlying architecture when implementing the functionality in the runtime.
+
+When you deploy a configured message flow, Partner Manager automatically translates the configurations used in the message flow and deploys and updates a set of applications necessary to orchestrate end-to-end B2B processing to the Mule runtime. These applications include:
+* Receive endpoint services
+* Document processing service
+* Replication service
+* Send endpoint services
+
+image::partner-manager-high-level-arch.png[Anypoint Partner Manager Architecture]
+
+[%header%autowidth.spread]
+|===
+|Callout |Name |Description
+|image:pm-arch-diagram-1.png[]
+|Partner Manager
+|MuleSoft hosted cloud control plane allowing users to configure trading partner profiles, build and deploy B2B integration message flows, and monitor B2B transactions activity.
+
+|image:pm-arch-diagram-2.png[]
+|Receive Endpoint Services
+|Service to receive inbound B2B or EDI messages from partners via communication protocols such as AS2 and SFTP or outbound application messages from your organization’s backend systems via HTTP, HTTPS, or SFTP.
```

Comment:
Please rephrase on the below lines:

Service to receive inbound B2B or EDI messages from partners via transport protocols such as AS2/SFTP, or outbound application messages from your organization’s backend systems via HTTP(s)/SFTP.